variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "aws_account_alternate_contacts" {
```

Comment:
Following along with the update to the variable name:

```suggestion
variable "account_alternate_contacts" {
```

Reply:
updated